threedi_modelchecker/config.py

```diff
@@ -2119,254 +2029,200 @@ def is_none_or_empty(col):
-    (
-        models.Interflow.hydraulic_conductivity_file,
-        interflow_filter,
-    ),
-    (
-        models.SimpleInfiltration.infiltration_rate_file,
-        infiltration_filter,
-    ),
-    (
-        models.SimpleInfiltration.max_infiltration_capacity_file,
-        infiltration_filter,
-    ),
-    (
-        models.GroundWater.groundwater_impervious_layer_level_file,
-        groundwater_filter,
-    ),
-    (
-        models.GroundWater.phreatic_storage_capacity_file,
-        groundwater_filter,
-    ),
-    (
-        models.GroundWater.equilibrium_infiltration_rate_file,
-        groundwater_filter,
-    ),
-    (
-        models.GroundWater.initial_infiltration_rate_file,
-        groundwater_filter,
-    ),
-    (
-        models.GroundWater.infiltration_decay_period_file,
-        groundwater_filter,
-    ),
-    (
-        models.GroundWater.groundwater_hydro_connectivity_file,
-        groundwater_filter,
-    ),
-    (models.GroundWater.leakage_file, groundwater_filter),
-    (models.GlobalSetting.initial_waterlevel_file, first_setting_filter),
-    (
-        models.GlobalSetting.initial_groundwater_level_file,
-        first_setting_filter & (models.GlobalSetting.groundwater_settings_id != None),
-    ),
-    (models.VegetationDrag.vegetation_height_file, vegetation_drag_filter),
-    (models.VegetationDrag.vegetation_stem_count_file, vegetation_drag_filter),
-    (models.VegetationDrag.vegetation_stem_diameter_file, vegetation_drag_filter),
-    (models.VegetationDrag.vegetation_drag_coefficient_file, vegetation_drag_filter),
+RASTER_COLUMNS = [
+    models.ModelSettings.dem_file,
+    models.ModelSettings.friction_coefficient_file,
+    models.Interception.interception_file,
+    models.Interflow.porosity_file,
+    models.Interflow.hydraulic_conductivity_file,
+    models.SimpleInfiltration.infiltration_rate_file,
+    models.SimpleInfiltration.max_infiltration_volume_file,
+    models.GroundWater.groundwater_impervious_layer_level_file,
+    models.GroundWater.phreatic_storage_capacity_file,
+    models.GroundWater.equilibrium_infiltration_rate_file,
+    models.GroundWater.initial_infiltration_rate_file,
+    models.GroundWater.infiltration_decay_period_file,
+    models.GroundWater.groundwater_hydraulic_conductivity_file,
+    models.GroundWater.leakage_file,
+    models.InitialConditions.initial_water_level_file,
+    models.InitialConditions.initial_groundwater_level_file,
+    models.VegetationDrag.vegetation_height_file,
+    models.VegetationDrag.vegetation_stem_count_file,
+    models.VegetationDrag.vegetation_stem_diameter_file,
+    models.VegetationDrag.vegetation_drag_coefficient_file,
 ]
 
 CHECKS += [
     GDALAvailableCheck(
-        error_code=700, level=CheckLevel.WARNING, column=models.GlobalSetting.dem_file
+        error_code=700, level=CheckLevel.WARNING, column=models.ModelSettings.dem_file
     )
 ]
+# TODO: check this check
 CHECKS += [
     RasterExistsCheck(
```

Comment:
I am a bit worried how this check will work if you chop off the folder name (which you did in https://github.com/nens/threedi-schema/pull/56)

Reply:
Fixed in 4acf4dd4228e83eeb6d2afb59a3a4089f13339b3